<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>570</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAbort">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>570</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Anuluj</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="surnameLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="peselLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Imie</string>
    </property>
   </widget>
   <widget class="QLabel" name="surnameLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nazwisko</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="cityCodeLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="cityLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="streetLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>510</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="peselLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PESEL</string>
    </property>
   </widget>
   <widget class="QLabel" name="sexLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Płeć</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hasło</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>E-mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="cityCodeLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kod pocztowy</string>
    </property>
   </widget>
   <widget class="QLabel" name="cityLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Miejscowosc</string>
    </property>
   </widget>
   <widget class="QLabel" name="streetLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ulica</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="birthDateLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="birthDateLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Data urodzenia</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hireDateLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hireDateLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Data zatrudnienia</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="loginLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="phoneLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Telefon</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="sexComboBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>pushButtonAdd</zorder>
   <zorder>pushButtonAbort</zorder>
   <zorder>nameLineEdit</zorder>
   <zorder>peselLineEdit</zorder>
   <zorder>nameLabel</zorder>
   <zorder>surnameLabel</zorder>
   <zorder>passwordLineEdit</zorder>
   <zorder>emailLineEdit</zorder>
   <zorder>cityCodeLineEdit</zorder>
   <zorder>cityLineEdit</zorder>
   <zorder>streetLineEdit</zorder>
   <zorder>peselLabel</zorder>
   <zorder>sexLabel</zorder>
   <zorder>passwordLabel</zorder>
   <zorder>emailLabel</zorder>
   <zorder>cityCodeLabel</zorder>
   <zorder>cityLabel</zorder>
   <zorder>streetLabel</zorder>
   <zorder>birthDateLineEdit</zorder>
   <zorder>birthDateLabel</zorder>
   <zorder>surnameLineEdit</zorder>
   <zorder>hireDateLineEdit</zorder>
   <zorder>hireDateLabel</zorder>
   <zorder>loginLineEdit</zorder>
   <zorder>loginLabel</zorder>
   <zorder>phoneLabel</zorder>
   <zorder>phoneLineEdit</zorder>
   <zorder>sexComboBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>surnameLineEdit</tabstop>
  <tabstop>sexComboBox</tabstop>
  <tabstop>birthDateLineEdit</tabstop>
  <tabstop>peselLineEdit</tabstop>
  <tabstop>hireDateLineEdit</tabstop>
  <tabstop>loginLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>phoneLineEdit</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>cityCodeLineEdit</tabstop>
  <tabstop>cityLineEdit</tabstop>
  <tabstop>streetLineEdit</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonAbort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
